tests/data/capabilities/kvm-loongarch64-domcaps.xml

```diff
@@ -0,0 +1,167 @@
+<domainCapabilities>
+  <path>/usr/bin/qemu-system-loongarch64</path>
+  <domain>kvm</domain>
+  <machine>virt</machine>
+  <arch>loongarch64</arch>
+  <vcpu max='256'/>
+  <iothreads supported='yes'/>
+  <os supported='yes'>
+    <enum name='firmware'/>
+    <loader supported='yes'>
+      <value>/obviously/fake/firmware1.fd</value>
+      <value>/obviously/fake/firmware2.fd</value>
```

Comment:
This looks like it comes right out of the libvirt test suite. Here's what the `<os>` element looks like on my running installation:

```xml
<os supported='yes'>
  <enum name='firmware'>
    <value>efi</value>
  </enum>
  <loader supported='yes'>
    <value>/usr/share/edk2/loongarch64/QEMU_EFI.fd</value>
    <enum name='type'>
      <value>rom</value>
      <value>pflash</value>
    </enum>
    <enum name='readonly'>
      <value>yes</value>
      <value>no</value>
    </enum>
    <enum name='secure'>
      <value>no</value>
    </enum>
  </loader>
</os>
```

Notice EFI support being advertised and fake paths being replaced with real ones.

Reply:
Ok, I did take the contents of this file from the libvirt test case, where does it normally come from? Is the content of the domainCapabilities I get with the virt-manager --no-fork --debug command more appropriate?